MAINTAINERS.md

```diff
@@ -23,6 +20,17 @@ Documentation Maintainers
 | Nikhil Gupta | nikhil550 | nikhilgupta | <negupta@us.ibm.com> |
 | Pam Andrejko | pamandrejko | pandrejko | <pama@ibm.com> |
 
+
+Retired Maintainers
+===================
+
+| Name | GitHub | Chat | email |
+|------|--------|------|-------|
+| Chris Ferris | christo4ferris | cbf | <chris.ferris@gmail.com> |
+| Jonathan Levi | hacera | JonathanLevi | <jonathan@hacera.com> |
```

Comment:
'hacera' is an organization, please let's use the 'hacera-jonathan' in the GitHub column

Reply:
Sure thing, if this PR gets approved I'll do another change to update Github user name.